libvirt/resource_libvirt_domain.go

```diff
@@ -45,6 +45,11 @@ func resourceLibvirtDomain() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
I think we should not use `forceNew` to true , instead should be false.

Is there any particular motivation for setting it to true? 

tia

Reply:
I couldn't find any way to update description on fly.
Without that, terraform will complain about missmatch description every run

Reply:
ok